pkg/sync/v9/translate.go

```
@@ -259,6 +255,18 @@ var translations = map[string][]struct {
 			NestedPath: jsonpath.MustCompile("$.subscriptionId"),
 			Template:   "{{ .ContainerService.Properties.AzProfile.SubscriptionID }}",
 		},
+		{
+			Path:       jsonpath.MustCompile("$.data.'config.yaml'"),
+			NestedPath: jsonpath.MustCompile("$.resourceGroupName"),
+			F: func(cs *api.OpenShiftManagedCluster, o interface{}) (interface{}, error) {
+				regex := *regexp.MustCompile(`/resource[gG]roups/([-\w\._\(\)]+)/`)
+				res := regex.FindStringSubmatch(cs.ID)
+				if len(res) != 2 {
+					return "_Empty", nil
```

why not return an error?